### PR TITLE
Ferdium.Ferdium.Nightly version 6.2.1-beta.1

### PR DESCRIPTION
**Pull request by Alphrag**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/85689)

### CODE REVIEW

File: manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Ferdium.Ferdium.Nightly
```
Comment (Trenly) [inline manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.installer.yaml:4]:
```suggestion
PackageIdentifier: Ferdium.Ferdium.Beta
```
This should be under the beta folder, not the nightly folder

File: manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Ferdium.Ferdium.Nightly
```
Comment (Trenly) [inline manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Ferdium.Ferdium.Beta
```
This should be under the beta folder, not the nightly folder

File: manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Ferdium.Ferdium.Nightly
```
Comment (Trenly) [inline manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.yaml:4]:
```suggestion
PackageIdentifier: Ferdium.Ferdium.Beta
```
This should be under the beta folder, not the nightly folder

File: manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Ferdium.Ferdium.Nightly
+PackageVersion: 6.2.1-beta.1
+PackageLocale: en-US
+Publisher: Ferdium Contributors
+PublisherUrl: https://ferdium.org
+PackageName: Ferdium Nightly
```
Comment (Trenly) [inline manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.locale.en-US.yaml:9]:
```suggestion
PackageName: Ferdium Beta
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Since Ferdium.Ferdium.Beta already exists as a package identifier, I believe it makes most sense for the beta versions to be there

**Review by ItzLevvie — APPROVED** [review]

**Comment by Alphrag** [comment]
> Since Ferdium.Ferdium.Beta already exists as a package identifier, I believe it makes most sense for the beta versions to be there

Yes, sorry for that. It appears that my script interpreted the last release as a Nightly instead of a Beta, which made the `wingetcreate update` base itself on the wrong manifest template. I have pushed changes that hopefully fix the manifests now.

**Pull request merged**